tests/acquire_shard_test.go

```diff
@@ -37,26 +37,25 @@ import (
 	"go.temporal.io/server/tests/testcore"
 )
 
-// AcquireShardFunctionalSuite is the base test suite for testing acquire shard.
-type AcquireShardFunctionalSuite struct {
+// AcquireShardSuiteBase is the base test suite for testing acquire shard.
+type AcquireShardSuiteBase struct {
 	testcore.FunctionalTestBase
 	logRecorder *logRecorder
 	logs        chan logRecord
 }
 
-func TestAcquireShardFunctionalSuite(t *testing.T) {
-	suite.Run(t, new(AcquireShardFunctionalSuite))
-}
-
 // SetupSuite sets up the test suite by setting the log recorder.
-func (s *AcquireShardFunctionalSuite) SetupSuite() {
-	s.logs = make(chan logRecord, 100)
+func (s *AcquireShardSuiteBase) SetupSuite() {
+	// Server startup happens in Setup_Suite_, but we don't listen on the log channel until the
+	// _test_ runs. So this needs to be big enough to buffer all of the server startup logs,
+	// which are currently about 190 lines (October 2024).
+	s.logs = make(chan logRecord, 2000)
 	s.logRecorder = newLogRecorder(s.logs)
```

Comment:
To save me from digging into the code, why don't we log these to stderr directly? Do we have assertions or eventually calls that inspect this buffer? Is this important enough to keep?

Reply:
The main mechanism of these tests is searching for strings in particular orders in the log buffer :face_with_diagonal_mouth: 

As I said on slack: I think these are pretty low-value tests (they're better suited to unit tests and we have unit tests covering this stuff). If they continue to be problematic I think we should delete them